src/App.tsx

```diff
@@ -10,33 +13,33 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+
+  const handelSelectTabId = (tab: Tab) => {
+    if (tab.id !== selectedTabId) {
+      setSelectedTabId(tab.id);
+    }
+  };
+
+  const selectTabTitle = () => {
```

Comment:
Consider using more descriptive function name. `selectTabTitle` could be changed to `getSelectedTabTitle` or `findSelectedTabTitle`. 🤔